lib/bike_brigade_web/live/campaign_signup_live/index.ex

```diff
@@ -22,13 +22,23 @@ defmodule BikeBrigadeWeb.CampaignSignupLive.Index do
     {:ok,
      socket
      |> assign(:page, :campaigns_signup)
-     |> assign(:page_title, "Campaign Signup List")
+     |> assign(:page_title, "Delivery Sign Up")
      |> assign(:current_week, current_week)
      |> assign(:campaign_task_counts, Delivery.get_total_tasks_and_open_tasks(current_week))
+     |> assign(:showing_urgent_campaigns, false)
      |> assign(:campaigns, campaigns)}
   end
 
   @impl true
+  def handle_params(%{"campaign_ids" => campaign_ids}, _url, socket) do
+    campaigns = fetch_campaigns(socket.assigns.current_week, campaign_ids: campaign_ids)
+
+    {:noreply,
+     socket
+     |> assign(:campaigns, campaigns)
+     |> assign(:showing_urgent_campaigns, true)}
```

Comment:
I think we should reset `showing_urgent_campaigns` to `false` in the other handle params so that it doesn't stick around if you navigate within the live view (which currently we don't have a button for, see below)